### PR TITLE
Release Beta Branch

### PR DESCRIPTION
**Pull request by bawelter**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -26,10 +27,31 @@ class _MyAppState extends State<MyApp> {
     String result;
     // Platform messages may fail, so we use a try/catch PlatformException.
     try {
-      database = await Database.initWithName("MyNewCouchbaseDB");
-      await database.saveDocumentWithId("test", Document({}));
-      int count = await database.count;
-      result = "Document Count: $count";
+      database = await Database.initWithName("spirit-bucket");
+      await database.saveDocumentWithId(DateTime.now().toIso8601String(), Document({}));
```
Comment (richard457) [inline example/lib/main.dart]: Looking at this code looks like we can save documents with an Id of our choice, is there a way this Id is returned in document response? I will check out but will be happy to know.

Comment (richard457) [reply example/lib/main.dart]: does this method saveDocumentWithId exist? I hunted down the code base could not find it.

Comment (richard457) [reply example/lib/main.dart]: I thought that this would be a solution to this https://github.com/SaltechSystems/couchbase_lite/issues/77

Comment (bawelter) [reply example/lib/main.dart]: You can specify the id when creating the mutable document.  The documentation is here https://pub.dev/documentation/couchbase_lite/latest/couchbase_lite/couchbase_lite-library.html

**Pull request merged**